src/app/(proper_react)/redesign/(public)/HeroImage.tsx

```diff
@@ -0,0 +1,795 @@
+        ></path>
+        <path
+          fill="url(#paint2_linear_1942_12276)"
+          d="M152.244 90.385l-4.005-2.294-9.144-5.236-.284-.163a4.889 4.889 0 00-4.836 0l-.284.165-12.858 7.364-.284.165a4.805 4.805 0 00-2.418 4.157v15.383a4.809 4.809 0 002.418 4.158l13.146 7.529a2.026 2.026 0 002.755-.731c.268-.46.34-1.008.201-1.521a1.988 1.988 0 00-.939-1.213l-12.858-7.363a1.373 1.373 0 01-.692-1.189V94.874c.001-.492.265-.946.692-1.19l2.276-1.304 10.568-6.064a1.394 1.394 0 011.379 0l12.862 7.366c.425.244.688.697.69 1.187v14.732a1.372 1.372 0 01-.693 1.189l-4.351 2.492-2.196-3.311a10.337 10.337 0 003.483-7.734c0-5.736-4.703-10.403-10.486-10.403s-10.487 4.667-10.487 10.403c0 5.736 4.704 10.402 10.487 10.402a10.518 10.518 0 003.54-.611l3.335 5.029c.052.077.109.15.17.22l.035.036c.07.076.147.146.229.209.02.016.038.033.06.048.09.065.185.123.284.174l.069.03c.088.04.179.073.273.101l.062.02c.106.027.214.044.323.052h.182c.061 0 .122-.007.182-.012h.084c.115-.016.228-.042.339-.077.029-.009.058-.022.088-.033.088-.032.173-.07.256-.114.018-.01.04-.015.058-.027l6.289-3.594a4.81 4.81 0 002.418-4.164V94.542a4.803 4.803 0 00-2.418-4.157zm-22.312 11.852a6.454 6.454 0 116.454 6.4 6.435 6.435 0 01-6.454-6.4z"
+        ></path>
+        <path
+          fill="url(#paint3_linear_1942_12276)"
+          d="M152.242 90.385l-4.005-2.294-5.587-3.197a4.308 4.308 0 00-4.16-.063l-2.795 1.485a1.396 1.396 0 011.38 0l12.862 7.366c.425.245.688.697.69 1.188v14.732a1.37 1.37 0 01-.693 1.189l-4.351 2.492.679 1.085a2.843 2.843 0 003.827.954l2.153-1.231a4.796 4.796 0 002.418-4.165V94.543a4.793 4.793 0 00-2.418-4.158z"
+        ></path>
+        <path
+          fill="url(#paint4_linear_1942_12276)"
+          d="M150.627 94.874v7.147h4.033v-7.478a4.793 4.793 0 00-2.418-4.158l-4.005-2.294-5.587-3.197a4.308 4.308 0 00-4.16-.063l-2.795 1.485a1.4 1.4 0 011.38 0l12.862 7.366c.426.246.689.7.69 1.192z"
+        ></path>
+        <path
+          fill="url(#paint5_linear_1942_12276)"
+          d="M150.625 103.171v6.431c0 .492-.265.945-.692 1.189l-4.351 2.491.678 1.086a2.846 2.846 0 003.828.954l2.153-1.232a4.795 4.795 0 002.418-4.164v-6.755h-4.034z"
+          opacity="0.9"
+        ></path>
+        <path
+          fill="url(#paint6_linear_1942_12276)"
+          d="M145.274 112.815l-1.887-2.845a10.504 10.504 0 01-3.463 2.057l2.007 3.028a89.272 89.272 0 003.343-2.24z"
+          opacity="0.9"
+        ></path>
+        <text
+          fill="#000"
+          style={{ whiteSpace: "pre" }}
+          fontFamily="var(--font-metropolis)"
```

Comment:
I replaced some embedded references to "Metropolis" as a font, to instead use the next/font variable - otherwise it wasn't able to actually find the relevant font.